### DISCUSSION TITLE
Can I apply for the Student Developer Pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22635] [New to GitHub]
<p>Hi everyone.</p>
<p>I am a high-school student (currently in grade 10). I would like to apply for the <a href="https://education.github.com/pack" rel="noopener nofollow ugc">Student Developer Pack</a>. At the <em>Join GitHub Education</em> page, I am asked to supply my <em>Graduation Year</em> which (of course) I cannot supply if I’m still in school. I am confused because in many other places the documentation mentions school qualifications. Given the above information, do I meet the requirements? I would really appreciate any help.</p>

**Comment by ghost** [comment, marked as answer]
<p><strong>DO NOT WORRY. I HAVE SORTED IT OUT. I AM SORRY FOR ANY TIME THAT I HAVE WASTED.</strong></p>

**Reply by divyanethmi** [reply]
i have requested as well but they don't give chance

**Comment by mpboom** [comment]
<p>If you share your solution here, other people might benefit from it. Then you turn a seamlessly “useless” post into something that others can use, too.</p>

**Comment by mpfaff** [comment]
<p>Hi. I am curious about the same thing. Could you please share your solution?</p>
